config/assessments.yml

```diff
@@ -132,7 +132,7 @@ parameters:
       wikiproject_prefix: 'Wikipedia:WikiProject '
       class:
         FA:
-          badge: 'b/bc/Featured_article_star.svg'
+          badge: '2/23/Symbol_star_FA.svg'
```

Comment:
This doesn't seem to be the standard icon used on the English Wikipedia. The icons were gathered from https://en.wikipedia.org/wiki/Wikipedia:Content_assessment, and these are the ones you see on talk pages, supplied by https://en.wikipedia.org/wiki/Template:WPBannerMeta. The classes below GA of course don't have official icons, so I just found suitable ones.